app/clarity.py

```diff
@@ -140,7 +142,12 @@ def scan_for_npc_names():
             name_addr = address + 48  # jump to name
             name = read_string(name_addr)
 
-            if data == "NPC":
+            if data == "NPC" or data == "MONSTER":
+                translated_names = ''
```

Comment:
You don't need to assign an empty string here as your conditional below is already assigning this variable. It won't end up unassigned with this logic.